src/components/movieDetail/MovieDetail.jsx

```diff
@@ -1,17 +1,42 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
 import './movieDetail.scss';
+import {useParams} from "react-router-dom";
+import axios from 'axios';
 
-const omdbMovieDetailResponse = "{\"Title\":\"Titanic\",\"Year\":\"1997\",\"Rated\":\"PG-13\",\"Released\":\"19 Dec 1997\",\"Runtime\":\"194 min\",\"Genre\":\"Drama, Romance\",\"Director\":\"James Cameron\",\"Writer\":\"James Cameron\",\"Actors\":\"Leonardo DiCaprio, Kate Winslet, Billy Zane\",\"Plot\":\"A seventeen-year-old aristocrat falls in love with a kind but poor artist aboard the luxurious, ill-fated R.M.S. Titanic.\",\"Language\":\"English, Swedish, Italian, French\",\"Country\":\"United States, Mexico\",\"Awards\":\"Won 11 Oscars. 125 wins & 83 nominations total\",\"Poster\":\"https:\/\/m.media-amazon.com\/images\/M\/MV5BMDdmZGU3NDQtY2E5My00ZTliLWIzOTUtMTY4ZGI1YjdiNjk3XkEyXkFqcGdeQXVyNTA4NzY1MzY@._V1_SX300.jpg\",\"Ratings\":[{\"Source\":\"Internet Movie Database\",\"Value\":\"7.9\/10\"},{\"Source\":\"Rotten Tomatoes\",\"Value\":\"87%\"},{\"Source\":\"Metacritic\",\"Value\":\"75\/100\"}],\"Metascore\":\"75\",\"imdbRating\":\"7.9\",\"imdbVotes\":\"1,141,554\",\"imdbID\":\"tt0120338\",\"Type\":\"movie\",\"DVD\":\"08 Jan 2002\",\"BoxOffice\":\"$659,363,944\",\"Production\":\"N\/A\",\"Website\":\"N\/A\",\"Response\":\"True\"}";
-let movieDetailObj = JSON.parse(omdbMovieDetailResponse);
-const MovieDetail = () => (
-    <div className="movie-info">
+
+function MovieDetail () {
+    const[movieDetailObj, setMovieDetailObj] = useState('');
+    var id = ImdbId();
+    useEffect(()=> {
+      getMovieDetailObj();   
+  
+    },[])
+    
+    const getMovieDetailObj= (props) => {
+      const movieApiurl = 'http://www.omdbapi.com';
+      var params = new URLSearchParams();
+      params.append("apikey", '18a01f17');
+      params.append("i",  id);
+  
+      var request = { params: params };
+  
+      axios.get(movieApiurl , request)
+      .then(resp => {
+         console.log("getMovieDetail response -> " + resp.data);
+         setMovieDetailObj(resp.data);
+      })
+      .catch(err => {
+            console.error("Error " + err);
+      })
+    }     
+
+    return (<div className="movie-info">
         <div className="background-black">
```

Comment:
I feel without black background, the page looks better and consistent with landing page.

Reply:
removed. initially added this to look better when landing page was not ready